ktorm-support-sqlite/src/main/kotlin/me/liuwj/ktorm/support/sqlite/SimpleSQLiteDialect.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package me.liuwj.ktorm.support.sqlite
+
+import me.liuwj.ktorm.database.Database
+import me.liuwj.ktorm.database.SqlDialect
+import me.liuwj.ktorm.database.prepareStatement
+import me.liuwj.ktorm.expression.SqlExpression
+import me.liuwj.ktorm.schema.Column
+import me.liuwj.ktorm.schema.IntSqlType
+import java.sql.PreparedStatement
+
+/**
+ * [SqlDialect] implementation for SQLite databases on plaforms with limited JDBC functionality. In particular
+ * Android and Robovm do not support calling prepareStatement with flags.
+ */
+open class SimpleSQLiteDialect : SQLiteDialect() {
```

Comment:
Is it necessary to add this class to Ktorm? SQLDroid supports `getGeneratedKeys()` now, so we don't need to execute the extra query manually anymore. The default strategy works fine.